src/server/api/schema.js

```diff
@@ -177,12 +178,10 @@ async function editCampaign(id, campaign, loaders, user, origCampaignRecord) {
     const convertedResponses = []
     for (let index = 0; index < cannedResponses.length; index++) {
       const response = cannedResponses[index]
-      const newId = await Math.floor(Math.random() * 10000000)
-      convertedResponses.push({
+      convertedResponses.push(_.omit({
```

Comment:
Two things here:
1. We don't need `lodash` or _.omit here, To do this in ES6, you could do this:
```
{ ...response, 
 campaign_id: id,
 id: undefined }
```
2. I'm a little nervous about us deleting and recreating canned_responses every time, but I guess if we've done that in the past, then there's no harm.

Reply:
👍